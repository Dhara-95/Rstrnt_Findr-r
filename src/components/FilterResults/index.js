import React from "react";

import { labels } from "../../Labels";

const FilterResults = () => {
  const { orderBy, nearestFilter, ratingFilter } = labels;

  return (
    <div className="flex justify-center p-4">
      <h2 className="text-grey m-4">{orderBy}</h2>
      <select
        name="filter"
        id="filter"
        onChange={(event) => {
          console.log(event.target.value);
        }}
      >
        <option value="nearest">{nearestFilter}</option>
        <option value="rating">{ratingFilter}</option>
      </select>
    </div>
  );
};

export { FilterResults };
