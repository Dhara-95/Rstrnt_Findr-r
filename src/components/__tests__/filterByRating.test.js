import React from "react";
import renderer from "react-test-renderer";

import { FilterResults } from "../FilterResults";

it("displays the dropdown menu", () => {
  const tree = renderer
    .create(
      <FilterResults
        updateRestaurantData={"updateRestaurantData"}
        businesses={"businesses"}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
