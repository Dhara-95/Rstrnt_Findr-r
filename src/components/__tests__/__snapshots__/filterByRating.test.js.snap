// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the dropdown menu 1`] = `
<div
  className="flex justify-center p-4 items-center w-1/2 mx-auto"
>
  <h2
    className="text-grey m-4"
  >
    Order by:
  </h2>
  <select
    className="rounded-full w-1/3 h-8"
    id="filter"
    name="filter"
    onChange={[Function]}
  >
    <option
      value="nearest"
    >
      Nearest to you
    </option>
    <option
      value="rating"
    >
      By rating
    </option>
  </select>
</div>
`;
